reverse_engineering/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "author": "Hackolade",
   "dependencies": {
-    "node-oracledb": "^4.2.0"
+    "oracledb": "^5.3.0"
```

Comment:
it seems that need to be built for each OS
https://oracle.github.io/node-oracledb/INSTALL.html#-2-quick-start-node-oracledb-installation